keps/sig-cli/859-kubectl-headers/README.md

```diff
@@ -257,11 +265,161 @@ included.
 
 ### Upgrade / Downgrade Strategy
 
-NA
+Not applicable. There are no cluster components affected by this feature.
+
 
 ### Version Skew Strategy
 
-NA
+Not applicable. There is nothing required of the API Server, so there
+can be no version skew.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+
+  This feature is enabled with the KUBECTL_COMMANDS_HEADER environment
```

Comment:
Making this opt-in at the client-level isn't particularly helpful, and the point is for cluster admins understand the context in which the apis are being invoked.  Making it opt-in at the cluster level might make sense?  What is the concern with making this opt-out?

Reply:
> Making this opt-in at the client-level isn't particularly helpful, and the point is for cluster admins understand the context in which the apis are being invoked and telling every user to set the environment variable isn't practical. Making it opt-in at the cluster level might make sense? What is the concern with making this opt-out?

Alpha features are opt-in.  Making an alpha feature opt-out would mean skipping alpha and going directly to beta.  There is some precedent for such a thing, but it's pretty uncommon.